lib/hammer_cli_foreman_remote_execution/job_invocation.rb

```diff
@@ -51,7 +53,31 @@ class InfoCommand < HammerCLIForeman::InfoCommand
         end
       end
 
+      def adapter
+        if option_id
+          :yaml
+        else
+          :base
+        end
+      end
+
       def extend_data(invocation)
+        if option_show_inputs?
+          invocation['template_invocations']&.each do |template|
+            input_values = template['template_invocation_input_values']
+            values_for_host = {}
+            input_values&.each do |input_value|
+              values_for_host[input_value['template_input_name']] = input_value['value']
+            end
+            hosts = invocation.dig('targeting', 'hosts')
+            host_index = hosts&.index { |h| h['id'] == template['host_id'] }
+            if host_index
+              invocation['targeting']['hosts'][host_index][:inputs] = values_for_host
+            else
+              invocation['targeting']['hosts'].append({ :inputs => values_for_host, 'name' => template['host_id'] })
```

Comment:
I would probably take `invocation['targeting']['hosts']` as the source of truth. So if there is an `invocation['template_invocations']` which doesn't have a matching host, it would get ignored.

Reply:
sure, changed!